concrete/tools/permissions/access/entity/types/submitter.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+defined('C5_EXECUTE') or die("Access Denied.");
+use Concrete\Core\Permission\Access\Entity\SubmitterEntity as SubmitterPermissionAccessEntity;
+
+if (\Loader::helper('validation/token')->validate('process')) {
+    $js = \Loader::helper('json');
```

Comment:
The JSON helper has been deprecated: we can directly use `json_encode` without any helper.